railties/test/engine/commands_test.rb

```diff
@@ -33,50 +33,48 @@ def test_runner_command_work_inside_engine
     assert_equal "test", output.strip
   end
 
-  if available_pty?
-    def test_console_command_work_inside_engine
-      primary, replica = PTY.open
-      cmd = "console"
-      spawn_command(cmd, replica, env: { "TERM" => "dumb" })
-      assert_output(">", primary)
-    ensure
-      primary.puts "quit"
-    end
+  def test_console_command_work_inside_engine
+    read, write = IO.pipe
```

Comment:
Maybe we want to keep using `PTY` for this test too, since it's opening an interactive Rails console?

Reply:
Yeah, I think it's probably more agnostic to IRB's behaviour (and ultimately a closer representation of user experience) to keep it here... though honestly pretty unlikely to ever come up as a concern either way.